src/logs/metadata_engine/jmespath.py

```diff
@@ -44,14 +44,23 @@ def _func_if(self, condition, if_true_expression, if_false_expression, node_scop
         else:
             return if_false_expression.visit(if_false_expression.expression, node_scope)
 
+
+    @functions.signature({'types': ['string', 'null']}, {'types': ['string', 'null']})
+    def _func_starts_with(self, search, suffix):
```

Comment:
`prefix` instead of `suffix`

Reply:
I saw it, I can fix it, but this is and exact clone from original jmespath implementation, apart from null's appearing.

Reply:
In their docs they have it named correctly: https://jmespath.org/specification.html#starts-with
I think we should name it correctly even if their implementation has got it wrong